src/datastore/postgres/schema/v5.ts

```diff
@@ -0,0 +1,26 @@
+import { IDatabase } from "pg-promise";
+
+// tslint:disable-next-line: no-any
+export async function runSchema(db: IDatabase<any>) {
+    // Create schema
+    await db.none(`
+    CREATE TABLE metrics_users (
```

Comment:
In general, I'd expect this to be a table of (user_id, last_active_ts)

Reply:
* `last_active_ts` wouldn't allow me to answer "active for at least N days in a given month". My plan is to JOIN this table with `metrics_user_room_activities` which has timestamps with the granularity of a day.